src/EventSubscriber/RedirectEventSubscriber.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\rules\EventSubscriber\RedirectEventSubscriber.
+ */
+
+namespace Drupal\rules\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Changes the response to a redirect, if a redirect rules action was executed .
+ */
+class RedirectEventSubscriber implements EventSubscriberInterface {
+
+  /**
+   * Checks is a redirect rules action was executed, and redirects to the
+   * provided url.
+   *
+   * @param \Symfony\Component\HttpKernel\Event\FilterResponseEvent $event
+   */
+  public function checkRedirectIssued(FilterResponseEvent $event) {
+    $request = $event->getRequest();
+    $redirect_url = $request->attributes->get('rulesRedirectActionUrl');
```

Comment:
Are we sure that we want to use camelCase here? I think we should use "_rules_redirect_action_url", core uses for example "_raw_variables" or "_optional_argument".